products/sle/main.pm

```diff
@@ -1316,6 +1316,13 @@ elsif (get_var('HPC')) {
         loadtest 'hpc/munge_slave'  if (check_var('HPC', 'munge_slave'));
     }
 }
+elsif (get_var('ROOTONLY')) {
+    load_boot_tests();
+    load_inst_tests();
+    load_reboot_tests();
+    loadtest "x11/xterm";
+    loadtest "x11/gnome_terminal";
```

Comment:
this PR please. We don't want to merge the PR today anyway because of RC2 validation